opacus/tests/privacy_engine_test.py

```diff
@@ -98,10 +108,19 @@ def _init_private_training(
         poisson_sampling: bool = True,
         clipping: str = "flat",
         grad_sample_mode="hooks",
+        opt_exclude_frozen=False,
     ):
         model = self._init_model()
         model = PrivacyEngine.get_compatible_module(model)
-        optimizer = torch.optim.SGD(model.parameters(), lr=self.LR, momentum=0)
+        optimizer = torch.optim.SGD(
+            [
```

Comment:
I would suggest something like this for readability
```python
model.parameters() 
if not opt_exclude_frozen 
else [p for p in model.parameters() if p.requires_grad]
```

Same in the `_init_vanilla_training `